sbt-plugin/src/main/scala/ch/epfl/scala/SubmitDependencyGraph.scala

```diff
@@ -96,12 +96,23 @@ object SubmitDependencyGraph {
       snapshot <- getSnapshot(httpResp)
     } yield {
       state.log.info(s"Submitted successfully as $snapshotUrl/${snapshot.id}")
+      setGithubOutputs(
+        "submission-id" -> snapshot.id.toString,
+        "submission-api-url" -> s"${snapshotUrl}/${snapshot.id}"
+      )
```

Comment:
One thought I had here was to write `snapshotJson` to a temp file and expose path to that file as an action output. That would allow downstream integration working with the JSON directly, instead of making an unnecessary call to the API. WDYT?